### ISSUE TITLE
Demo site not working

### ISSUE THREAD

**Issue by fishnyc22** [JeremyEnglert/JointsWP-Documentation#19]
Just a heads up your demo site is not loading properly. https://jointswp.com/demo/
Cheers,

**Comment by lblanton** [comment]
> Just a heads up your demo site is not loading properly. https://jointswp.com/demo/
> Cheers,

Yeah, it looks like the stylesheets are missing as they are returning 404 errors.
